mix.exs

```diff
@@ -57,8 +58,8 @@ defmodule Phoenix.Mixfile do
 
   defp deps do
     [
-      {:cowboy, "~> 1.0", optional: true},
-      {:plug, "~> 1.3.3 or ~> 1.4"},
+      cowboy_dep(System.get_env("COWBOY_VERSION")),
+      {:plug, "~> 1.5.0-rc.0", override: true},
```

Comment:
@josevalim @chrismccord If we are happy with specifying this version for now then this can be merged.

I'll do a new PR updating the generator so we can release a new archive and people can begin trying this out.